ovs_performance.py

```diff
@@ -2805,6 +2818,9 @@ def main():
     parser.add_argument("--stream-list", metavar="LIST",
                         help="List of stream sizes to test", type=str,
                         default=DEFAULT_STREAM_LIST)
+    parser.add_argument("--stream-limit", metavar="MAX-FLOWS",
```

Comment:
This option is not needed and also does not make sense as there is the --stream-list option. Please use that instead.